plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/action/DeleteQueryGroupResponse.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.action;
+
+import org.opensearch.cluster.metadata.QueryGroup;
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Response for delete QueryGroup
+ *
+ * @opensearch.experimental
+ */
+public class DeleteQueryGroupResponse extends ActionResponse implements ToXContent, ToXContentObject {
+    private final List<QueryGroup> queryGroups;
```

Comment:
Similar to get API, should not be a list. The QueryGroup response has to be exactly the one deleted as part of this delete request